<!DOCTYPE html>
<!--[if IEMobile 7]><html class="iem7"  lang="en" dir="ltr"><![endif]-->
<!--[if lte IE 6]><html class="lt-ie9 lt-ie8 lt-ie7"  lang="en" dir="ltr"><![endif]-->
<!--[if (IE 7)&(!IEMobile)]><html class="lt-ie9 lt-ie8"  lang="en" dir="ltr"><![endif]-->
<!--[if IE 8]><html class="lt-ie9"  lang="en" dir="ltr"><![endif]-->
<!--[if (gte IE 9)|(gt IEMobile 7)]><!--><html  lang="en" dir="ltr" prefix="content: http://purl.org/rss/1.0/modules/content/ dc: http://purl.org/dc/terms/ foaf: http://xmlns.com/foaf/0.1/ og: http://ogp.me/ns# rdfs: http://www.w3.org/2000/01/rdf-schema# sioc: http://rdfs.org/sioc/ns# sioct: http://rdfs.org/sioc/types# skos: http://www.w3.org/2004/02/skos/core# xsd: http://www.w3.org/2001/XMLSchema#"><!--<![endif]-->

<head>
  <meta charset="utf-8" />
<link rel="shortlink" href="48.html" />
<link rel="canonical" href="48.html" />
<meta name="description" content="Below is the list of so far accepted posters. There is still possibility of submitting late posters, however there are only few spaces left and we cannot guarantee the presence of late poster abstracts in the abstract book." />
<meta content="Accepted Posters" about="/accepted-posters" property="dc:title" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
  <title>Accepted Posters | RECOMB 2015</title>

      <meta name="MobileOptimized" content="width">
    <meta name="HandheldFriendly" content="true">
    <meta name="viewport" content="width=device-width">
    <meta http-equiv="cleartype" content="on">

  <style>@import url("../modules/system/system.base.css%3Fn7bq9j.css");</style>
<style>@import url("../modules/field/theme/field.css%3Fn7bq9j.css");
@import url("../modules/node/node.css%3Fn7bq9j.css");
@import url("../modules/user/user.css%3Fn7bq9j.css");
@import url("../sites/all/modules/views/css/views.css%3Fn7bq9j.css");</style>
<style>@import url("../sites/all/modules/ckeditor/ckeditor.css%3Fn7bq9j.css");
@import url("../sites/all/modules/ctools/css/ctools.css%3Fn7bq9j.css");</style>
<style>@import url("../sites/all/themes/recomb/css/styles.css%3Fn7bq9j.css");</style>
  <script src="../misc/jquery.js%3Fv=1.4.4"></script>
<script src="../misc/jquery.once.js%3Fv=1.2"></script>
<script src="../misc/drupal.js%3Fn7bq9j"></script>
<script src="../sites/all/modules/google_analytics/googleanalytics.js%3Fn7bq9j"></script>
<script>var _gaq = _gaq || [];_gaq.push(["_setAccount", "UA-49675284-1"]);_gaq.push(["_trackPageview"]);(function() {var ga = document.createElement("script");ga.type = "text/javascript";ga.async = true;ga.src = ("https:" == document.location.protocol ? "https://ssl" : "http://www") + ".google-analytics.com/ga.js";var s = document.getElementsByTagName("script")[0];s.parentNode.insertBefore(ga, s);})();</script>
<script>jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"recomb","theme_token":"GZ_NojOyQ-mxHFKS9nMz6ZyfJ-LFrFPaA4eAILYk2uA","js":{"misc\/jquery.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/google_analytics\/googleanalytics.js":1,"0":1},"css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"modules\/field\/theme\/field.css":1,"modules\/node\/node.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/views\/css\/views.css":1,"sites\/all\/modules\/ckeditor\/ckeditor.css":1,"sites\/all\/modules\/ctools\/css\/ctools.css":1,"sites\/all\/themes\/recomb\/system.menus.css":1,"sites\/all\/themes\/recomb\/system.messages.css":1,"sites\/all\/themes\/recomb\/system.theme.css":1,"sites\/all\/themes\/recomb\/css\/styles.css":1}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls|xml|z|zip"}});</script>
      <!--[if lt IE 9]>
    <script src="/sites/all/themes/zen/js/html5-respond.js"></script>
    <![endif]-->
  </head>
<body class="html not-front not-logged-in one-sidebar sidebar-first page-node page-node- page-node-48 node-type-article section-accepted-posters" >
      <p id="skip-link">
      <a href="48.html#main-menu" class="element-invisible element-focusable">Jump to navigation</a>
    </p>
      <div id="container">
  <div id="page-top">
    <div id="page-top-inner">
        <div class="region region-highlighted">
    <div id="block-block-2" class="block block-block first last odd">

      
  <h2><img alt="Warsaw, Poland, April 12-15, 2015" src="../sites/default/files/recomb-date-place.png" style="height:27px; width:392px" /></h2>

</div>
  </div>
    </div>
  </div>
  <div id="page-header">
    <div id="page-header-inner">
      <header class="header" id="header" role="banner">

                  <a href="../index.html" title="Home" rel="home" class="header__logo" id="logo"><img src="../sites/default/files/recomb-logo_0.png" alt="Home" class="header__logo-image" /></a>
        
                  <div class="header__name-and-slogan" id="name-and-slogan">
                          <h1 class="header__site-name" id="site-name">
                <a href="../index.html" title="Home" class="header__site-link" rel="home"><span>RECOMB 2015</span></a>
              </h1>
            
                      </div>
        
                  <div class="header__region region region-header">
    <div id="block-block-3" class="block block-block first odd">

      
  <p><img alt="19th Annual International Conference on Research in Computational Molecular Biology" src="../sites/default/files/recomb-slogan_1.png" style="height:49px; width:263px" /></p>

</div>
<div id="block-block-4" class="block block-block last even">

      
  <p><img alt="" src="../sites/default/files/recomb_animacja_32colors.gif" style="height:290px; width:908px" /></p>

</div>
  </div>
      </header>
    </div>
  </div>

  <div id="page">
    <div id="main">

      <div id="content" class="column" role="main">
                <a id="main-content"></a>
                          <h1 class="page__title title" id="page-title">Accepted Posters</h1>
                                                        


<article class="node-48 node node-article node-promoted view-mode-full clearfix" about="/accepted-posters" typeof="sioc:Item foaf:Document">

      <header>
                  
              <p class="submitted">
                    <span property="dc:date dc:created" content="2015-02-27T13:15:33+01:00" datatype="xsd:dateTime" rel="sioc:has_creator">Submitted by <span class="username" xml:lang="" about="/users/bartek" typeof="sioc:UserAccount" property="foaf:name" datatype="">bartek</span> on <time pubdate datetime="2015-02-27T13:15:33+01:00">Fri, 02/27/2015 - 13:15</time></span>        </p>
      
          </header>
  
  <div class="field field-name-body field-type-text-with-summary field-label-hidden"><div class="field-items"><div class="field-item even" property="content:encoded"><p>Below is the list of so far accepted posters. There is still possibility of submitting late posters, however there are only few spaces left and we cannot guarantee the presence of late poster abstracts in the abstract book.</p>
<table cellspacing="0" border="0"><tbody><tr><td>178</td>
<td>Soo Young Cho, Young Seek Lee, Soojun Park and Je Kyung Seong.<strong> Target Gene Selection System for Human Disease Related Mouse Models</strong></td>
</tr><tr><td>184</td>
<td>Osval Antonio Montesinos López and Emeterio Franco Pérez.<strong> Threshold models for genome-enabled prediction of ordinal categorical traits in plant breeding</strong></td>
</tr><tr><td>190</td>
<td>Chen-Hsiang Yeang, Yen-Fu Chen, Kai-Neng Chuang, Chih-Hsu Lin and Shery Yen.<strong> Alternative definitions of UGA codon regulated by translational regulatory factors and selenium</strong></td>
</tr><tr><td>217</td>
<td>David Seifert, Francesca Di Giallonardo, Karin J. Metzner, Huldrych F. Günthard and Niko Beerenwinkel.<strong> A Framework for Inferring Fitness Landscapes of Patient-Derived Viruses Using Quasispecies Theory</strong></td>
</tr><tr><td>219</td>
<td>Mani P. Grover.<strong> Identification of novel therapeutics for complex diseases from genome-wide association data</strong></td>
</tr><tr><td>220</td>
<td>Vivekanand A, Reshmi G and M. Radhakrishana Pillai.<strong> miRBP: An architecture to find competitive interaction between microRNA and RNA binding proteins</strong></td>
</tr><tr><td>221</td>
<td>Leszek Pryszcz and Toni Gabaldón.<strong> Redundans: an assembly pipeline for highly heterozygous genomes</strong></td>
</tr><!--<tr><td>222</td>
<td>Pao-Yang Chen.<strong> A comprehensive pipeline for the epigenomic data analysis of genome wide DNA methylation</strong></td>
</tr>
<p>--><tr><td>223</td>
<td>Marius Mihasan.<strong> Molecular classification and comparative analysis of Arthrobacter genus plasmids</strong></td>
</tr><tr><td>224</td>
<td>Szymon Wasik and Jacek Blazewicz.<strong> Trochilus – a platform for evaluation of algorithms that find values of real parameters</strong></td>
</tr><tr><td>225</td>
<td>Marcin Borowski, Tomasz Głowacki and Piotr Formanowicz.<strong> Heuristic approach for peptide assembly problem</strong></td>
</tr><tr><td>226</td>
<td>Natalia Szóstak, Agnieszka Rybarczyk, Marta Szachniuk and Jacek Błażewicz.<strong> Sorting signals targeting mRNA into hepatic extracellular vesicles</strong></td>
</tr><tr><td>227</td>
<td>Marzena Dolbniak, Roman Jaksik, Joanna Rzeszowska-Wolny and Krzysztof Fujarewicz.<strong> A model for the effects of ionizing radiation on microRNA-mediated regulation of mRNA levels in cells</strong></td>
</tr><tr><td>229</td>
<td>Maciej Antczak, Tomasz Zok, Martin Riedel, David Nebel, Piotr Lukasiak, Marta Szachniuk, Thomas Villmann and Jacek Blazewicz.<strong> Accurate prediction of nucleotide conformations</strong></td>
</tr><tr><td>230</td>
<td>Eugene Bolotin, Chenxi Chen, Apoorva Dhabalia and Jane Su.<strong> Illumina-NextBio Body Atlas now features RNA-seq data.</strong></td>
</tr><tr><td>231</td>
<td>Rafal Zaborowski, Torgeir Hvidsten and Bartek Wilczyński.<strong> Genes co-localization in topologically associating domains indicates higher co-expression</strong></td>
</tr><tr><td>232</td>
<td>Marta Wiśniewska, Adam Liwo and Mariusz Makowski.<strong> Analytical Formulas for the Potentials of Mean Force of the Interaction of O-phosphorylated and Hydrophobic Amino-Acid Side Chains in Water</strong></td>
</tr><tr><td>233</td>
<td>Benedikt Zacher, Michael Lidschreiber, Patrick Cramer, Julien Gagneur and Achim Tresch.<strong> Annotation of genomics data by the bidirectional hidden Markov model unveils variations in Pol II transcription cycle</strong></td>
</tr><tr><td>234</td>
<td>Paweł Bednarz and Bartek Wilczynski.<strong> Integrative computational model for tissue specific gene regulation</strong></td>
</tr><tr><td>235</td>
<td>Nadav Brandes and Michal Linial.<strong> Overlapping Genes in Viruses: Seeking a Unified Theory</strong></td>
</tr><tr><td>236</td>
<td>Jakub Rydzewski, Wieslaw Nowak and Giuseppe Nicosia.<strong> Inferring normal and pathological states in cortical neuron circuit model by Sensitivity Analysis algorithms</strong></td>
</tr><tr><td>237</td>
<td>Byungwook Lee.<strong> Protein sequence comparison using domain-based approaches</strong></td>
</tr><tr><td>238</td>
<td>Jonathan Crowther, Yanyan Cai, Layka Abbasi, Stein Aerts and Anna Sablina.<strong> The Role of Chromosomal Deletions as Prognostic Markers of Drug Sensitivity</strong></td>
</tr><tr><td>239</td>
<td>Vân Anh Huynh-Thu and Guido Sanguinetti.<strong> Combining tree-based and dynamical systems for the inference of gene regulatory networks</strong></td>
</tr><tr><td>240</td>
<td>Albert Herencsár, Krzysztof Piecuch and Broňa Brejová.<strong> An Improved Algorithm for Ancestral Gene Order Reconstruction</strong></td>
</tr><tr><td>241</td>
<td>Marta Kulik, Takaharu Mori, Yuji Sugita and Joanna Trylska.<strong> The Dynamics of the N1 Riboswitch Interacting with Aminoglycoside Antibiotics</strong></td>
</tr><tr><td>242</td>
<td>Przemysław Gagat, Andrzej Bodył and Paweł Mackiewicz.<strong> Bioinformatics studies of protein translocons in photosynthetic organelles of the testate amoeba Paulinella chromatophora</strong></td>
</tr><tr><td>243</td>
<td>Karel Břinda, Valentina Boeva and Gregory Kucherov.<strong> RNF: a method and tools to evaluate NGS read mappers</strong></td>
</tr><tr><td>244</td>
<td>Julian Zubek, Adam Boniecki, Maciej Mnich, Marcin Tatjewski, Subhadip Basu and Dariusz Plewczynski.<strong> Multi-level machine learning prediction of protein-protein interactions</strong></td>
</tr><tr><td>245</td>
<td>Michal Burdukiewicz, Piotr Sobczyk, Paweł Błażej and Paweł Mackiewicz.<strong> Predicting eukaryotic signal peptides using hidden Markov models</strong></td>
</tr><tr><td>246</td>
<td>Maciej Baranowski and Stanisław Ołdziej.<strong> Substrate recognition by Hsp40 chaperones - molecular dynamics studies.</strong></td>
</tr><tr><td>247</td>
<td>Piotr Sobczyk, Michał Burdukiewicz, Chris Lauber and Paweł Mackiewicz.<strong> Quick Permutation Test: feature filtering of n-gram data</strong></td>
</tr><tr><td>248</td>
<td>Piotr Posacki.<strong> Monte Carlo simulations of mammalian sex chromosomes evolution.</strong></td>
</tr><tr><td>249</td>
<td>Maciej Jasiński, Anna Górska and Joanna Trylska.<strong> Analysis of RNA thermodynamics using Motif Identifier for Nucleic Acid Trajectory</strong></td>
</tr><tr><td>250</td>
<td>Małgorzata Grabińska, Pawel Blazej and Paweł Mackiewicz.<strong> Application of Monte Carlo simulations and Metropolis-Hastings algorithm in analysis of mutation accumulation in three codon positions of protein coding sequences</strong></td>
</tr><tr><td>251</td>
<td>Magdalena Żulpo and Malgorzata Kotulska.<strong> Modeling hypothetical amyloid pores</strong></td>
</tr><tr><td>252</td>
<td>Pascal St-Onge, Mathieu Lajoie, Patrick Beaulieu, Simon Drouin, Jasmine Healy and Daniel Sinnett.<strong> A Framework For High Throughput Sequencing Analysis Of Childhood Leukemia</strong></td>
</tr><tr><td>253</td>
<td>Dmitry Podolsky, Ivan Molodtcov, Alexander Zenin, Valeria Kogan, Andrey Tarkhov, Leonid Menshikov, Robert J Shmookler Reis and Peter Fedichev.<strong> Critical dynamics of gene networks is behind ageing and Gompertz law</strong></td>
</tr><tr><td>254</td>
<td>Tomasz Zok, Maciej Antczak, Mariusz Popenda, Piotr Lukasiak, Ryszard Adamiak, Jacek Blazewicz and Marta Szachniuk.<strong> RNApdbee - secondary structure retrieval from knotted and unknotted RNA structures</strong></td>
</tr><tr><td>255</td>
<td>Insu Jang and Byungwook Lee.<strong> An introduction to miRseqViewer: Visualization tool for microRNA analysis</strong></td>
</tr><tr><td>256</td>
<td>Małgorzata Wnętrzak, Paweł Błażej and Paweł Mackiewicz.<strong> Optimality of the canonical genetic code under multiobjective and mutually exclusive criteria</strong></td>
</tr><tr><td>257</td>
<td>Aleksandra Kroczak, Adam Urantówka and Paweł Mackiewicz.<strong> Influence of methods and molecular markers’ selection on inferring phylogenetic relationships in the example of parrots from tribe Arini</strong></td>
</tr><tr><td>258</td>
<td>Ilona Grabowicz and Bartek Wilczyński.<strong> Outlier sample detection and exclusion improves differential gene expression analysis</strong></td>
</tr><tr><td>259</td>
<td>Amit Frishberg and Irit Gat-Viks.<strong> Computational prediction of changes in quantities of immune cell types</strong></td>
</tr><tr><td>260</td>
<td>Paweł Woźniak and Małgorzata Kotulska.<strong> AMYload – web service dedicated to amyloidogenic proteins.</strong></td>
</tr><tr><td>261</td>
<td>Klev Diamanti, Husen M. Umer, Marcin Kruczyk, Marco Cavalli, Claes Wadelius and Jan Komorowski.<strong> Detection of putative regulatory regions and genomic signal interactions</strong></td>
</tr><tr><td>262</td>
<td>Avital Brodt, Maya Botzman, Eyal David and Irit Gat-Viks.<strong> Dissecting Dynamic Genetic Variation that Controls Temporal Gene Response in Yeast</strong></td>
</tr><tr><td>263</td>
<td>Uma Shankavaram, Orieta Celiku, Xiang Deng, Shuping Zhao, Anita Tandle and Kevin Camphausen.<strong> Drug Response Modeling of Cancer using Network Pharmacology</strong></td>
</tr><tr><td>264</td>
<td>Tom Harel and Irit Gat-Viks.<strong> Combining genetic polymorphisms with inherited variation in gene expression to explain organismal physiological traits:</strong></td>
</tr><tr><td>265</td>
<td>Ewa Golas, Adam Liwo, Harold A. Scheraga, Cezary Czaplewski and Jaroslaw Marszalek.<strong> The ClpB disaggregase: molecular dynamics simulation of a re-modelling machine</strong></td>
</tr><tr><td>266</td>
<td>Anna Gogolinska and Wieslaw Nowak.<strong> Petri Nets as a Novel Representation of Biomolecular Simulations Data</strong></td>
</tr><tr><td>267</td>
<td>Magdalena Ochab and Krzysztof Puszynski.<strong> Mathematical model of the p53-dependent apoptotic pathway abnormalities in the non-small cell lung cancer</strong></td>
</tr><tr><td>268</td>
<td>Agnieszka G. Lipska, Steve R. Seidman, Harold A Scheraga and Adam Liwo.<strong> Implementation of hydrodynamic interactions in molecular dynamics simulations with the UNRES force field</strong></td>
</tr><tr><td>269</td>
<td>Dorota Formanowicz, Agnieszka Rybarczyk, Marcin Radom and Piotr Formanowicz.<strong> A role of inflammation and immunity in essential hypertension and cardiovascular disease – modeled and analyzed using Petri nets</strong></td>
</tr><tr><td>270</td>
<td>Dorota Formanowicz, Marcin Radom, Agnieszka Rybarczyk and Piotr Formanowicz.<strong> The importance of iron in the process of atherosclerosis modeled by Petri net based approach</strong></td>
</tr><tr><td>271</td>
<td>Vladimír Boža, Jakub Jursa, Broňa Brejová and Tomas Vinar.<strong> Efficient Indexing of Read Collections for Likelihood Computation</strong></td>
</tr><tr><td>272</td>
<td>Mitra Kabir, Andrew J. Doig and Kathryn E. Hentges.<strong> The relationship between duplication and essentiality for mammalian genes</strong></td>
</tr><tr><td>273</td>
<td>Agnieszka Karczyńska, Paweł Krupa, Magdalena Mozolewska and Adam Liwo.<strong> Assessment of the resolution of the UNRES force field with structure-based restraints.</strong></td>
</tr><tr><td>274</td>
<td>Mark Rogers, Hashem Shihab, Julian Gough, Matthew Mort, David Cooper, Ian Day, Tom Gaunt and Colin Campbell.<strong> Predicting the functional impact of genetic sequence variation in human disease</strong></td>
</tr><tr><td>275</td>
<td>Bence Szalai, Susanne Prokop, Miklós Cserző, Péter Várnai and László Hunyady.<strong> Co-evolution and co-expression based analysis and prediction of G Protein-coupled receptor heterodimerization</strong></td>
</tr><tr><td>276</td>
<td>Robert Schöpflin and Martin Vingron.<strong> Identification of cis-regulatory elements from chromosome conformation capture data</strong></td>
</tr><tr><td>277</td>
<td>Karol Nienałtowski and Michał Komorowski.<strong> Efficient experimental design for systems biology dynamical models</strong></td>
</tr><tr><td>278</td>
<td>Krzysztof Gogolewski, Michał Startek, Dariusz Grzebelus, Arnaud Le Rouzic and Anna Gambin.<strong> The potential role of transposable elements as evolutionary helpers in sexual populations - the computational model.</strong></td>
</tr><tr><td>279</td>
<td>Wayne Dawson, Michal Boniecki and Janusz Bujnicki.<strong> An entropy model for ranking structures and measuring flexibility in 3D RNA simulations using SimRNA</strong></td>
</tr><tr><td>280</td>
<td>Sahar Ansari, Michele Donato and Sorin Draghici.<strong> Inferring disease mechanisms from multiple gene expression datasets using PPI</strong></td>
</tr><tr><td>281</td>
<td>Michal J. Dabrowski, Michal Draminski, Klev Diamanti, Jacek Koronacki and Jan Komorowski.<strong> Discovering interdependencies in genomic data and the case of immune system genes</strong></td>
</tr><tr><td>282</td>
<td>Marcin Tatjewski, Julian Zubek, Marcin Kierczak and Dariusz Plewczyński.<strong> Towards discovering key factors in prediction of post-translational modifications</strong></td>
</tr><tr><td>283</td>
<td>Dennis Wang, Sara Dempster and Nirmal Keshava.<strong> Approaches for integrative clustering of cancer cell lines for evaluation of therapeutic response</strong></td>
</tr><tr><td>284</td>
<td>Luis Zapata, Hana Susak, Oliver Drechsel and Stephan Ossowski.<strong> Signatures of positive selection reveal driver genes across multiple cancer types</strong></td>
</tr><tr><td>285</td>
<td>Shamba Mondal, Yiliang Wei, David Arnosti and Bartek Wilczynski.<strong> Computational analysis of sequence motifs for discriminating different ChIP-exo profiles of related RBF proteins</strong></td>
</tr><tr><td>286</td>
<td>Peter Hansen, Jochen Hecht, Daniel Ibrahim, Alexander Krannich and Peter N. Robinson.<strong> Q - A saturation-based peak caller for reproducible analysis of transcription factor binding sites</strong></td>
</tr><tr><td>287</td>
<td>Xi Gao and Tomasz Arodz.<strong> Combining biological networks with ensemble learning for analyzing molecular profiles</strong></td>
</tr><tr><td>288</td>
<td>Michelle Flowers, Kristin Helling, Jing Zhang and John Karro.<strong> POSTER: Identifying strand-symmetry reversal in the human genome</strong></td>
</tr><tr><td>289</td>
<td>Dorota Mackiewicz and Stanisław Cebrat.<strong> Non-random distribution of recombination hotspot motifs in human genome</strong></td>
</tr><tr><td>290</td>
<td>Mina Maleki, Mohammad Haj Dezfulian and Luis Rueda.<strong> Computational modeling of the effect of amino acid substitutions on the formation of Ubiquitin-based complexes</strong></td>
</tr><tr><td>291</td>
<td>Bartłomiej Zaborowski, Dawid Jagieła, Adam Sieradzan, Cezary Czaplewski, Anna Hałabis, Agnieszka Lewandowska, Wioletta Żmudzińska, Stanisław Ołdziej and Adam Liwo.<strong> A maximum-likelihood approach to force-field training for protein structure prediction and folding simulations</strong></td>
</tr><tr><td>292</td>
<td>Abed Alkhateeb, Siva Reddy, Iman Rezaeian, Urvashi Katiyar, Dina Maskoni, John Kelly, Dora Cavallo-Medved, Lisa Porter and Luis Rueda.<strong> Analysis of Novel mRNA Transcripts in Prostate Cancer</strong></td>
</tr><tr><td>293</td>
<td>Adam Sieradzan, Nevena Litova, Adam Liwo and Antti Niemi.<strong> How do loops form in proteins? A key to protein folding?</strong></td>
</tr><tr><td>294</td>
<td>Agata Charzyńska and Anna Gambin.<strong> Global sensitivity analysis based on knn-estimators for information theoretic measures</strong></td>
</tr><tr><td>295</td>
<td>Marcin Pacholczyk, Karolina Smolińska and Marek Kimmel.<strong> Computational method for modeling and testing of transcription factor binding sites</strong></td>
</tr><tr><td>296</td>
<td>Vladimir Shchur and Richard Durbin.<strong> Tree consistent PBWT and their application to reconstructing Ancestral Recombination Graphs and demographic inference</strong></td>
</tr><tr><td>297</td>
<td>Nathanael Fillmore, Bo Li, Yongsheng Bai, Mike Collins, James Thompson, Ron Stewart and Colin Dewey.<strong> Evaluation of de novo transcriptome assemblies from RNA-Seq data</strong></td>
</tr><tr><td>298</td>
<td>Aleksander Jankowski, Jerzy Tiuryn and Shyam Prabhakar.<strong> MOCCA: Accurate identification of transcription factor binding sites from DNase-seq data</strong></td>
</tr><tr><td>299</td>
<td>Bogumil Konopka, Rafal Roszak and Malgorzata Kotulska.<strong> Protein Contact Ontology - a tool for annotation of protein residue-residue contacts</strong></td>
</tr><tr><td>300</td>
<td>Marcin Sobieraj, Marek Kalinowski and Bogdan Lesyng.<strong> ALGORYTHMIC APPROACH BASED ON QUATERNION ALGEBRA TO ANALYZE CAUSALITY OF STRUCTURAL CHANGES IN BIOMOLECULAR SYSTEMS</strong></td>
</tr><tr><td>301</td>
<td>Atefeh Taherian Fard, Sriganesh Srihari, Jessica C Mar and Mark A Ragan.<strong> Modeling the landscape of cellular development using the Hopfield network</strong></td>
</tr><tr><td>302</td>
<td>Shaun D Jackman, Karthika Raghavan, Benjamin P Vandervalk, Daniel Paulino, Justin Chu, Hamid Mohamadi, Anthony Raymond, René L Warren and Inanç Birol.<strong> Scaling ABySS to longer reads using spaced k-mers and Bloom filters</strong></td>
</tr><tr><td>303</td>
<td>Mateusz Krzysztof Łącki and Anna Gambin.<strong> Localised Fine Structure in Mass Spectrometry</strong></td>
</tr><tr><td>304</td>
<td>Witold Rudnicki, Szymon Migacz, Antoni Rościszewski, Andrzej Sułecki, Łukasz Grad, Magdalena Zaremba and Paweł Tabaszewski.<strong> Feature Selection Server</strong></td>
</tr><tr><td>305</td>
<td>Maciej Dziubinski, Pawel Daniluk and Bogdan Lesyng.<strong> ResiCon: a web service for the identification of dynamic domains, hinges and interfacial regions in proteins</strong></td>
</tr><tr><td>306</td>
<td>Ewa Szczurek, Tyll Krüger and Niko Beerenwinkel.<strong> The bottleneck of metastasis formation: insights from a stochastic model</strong></td>
</tr><tr><td>307</td>
<td>Karel Břinda, Maciej Sykulski and Gregory Kucherov.<strong> Spaced seeds improve metagenomic classification</strong></td>
</tr><tr><td>308</td>
<td>Weronika Sikora-Wohlfeld, Abhi K. Basu, Monica Martinez-Canales and Atul J. Butte.<strong> Utilizing the Intel Genomics Reference Architecture for Analyzing Genome Sequencing Data</strong></td>
</tr><tr><td>309</td>
<td>Nadav Rappoprt and Michal Linial.<strong> Entropy-driven partitioning of the hierarchical protein space</strong></td>
</tr><tr><td>310</td>
<td>Beata Sokołowska, Krystiana Krzyśko, Irena Niebroj-Dobosz, Marta Hallay-Suszek, Łukasz Charzewski, Agnieszka Madej-Pilarczyk, Michał Marchel, Irena Hausmanowa-Petrusewicz and Bogdan Lesyng.<strong> The statistical and MM/MD molecular modeling of MMP-TIMP system can be helpful in distinction of EDMD forms and healthy controls</strong></td>
</tr><tr><td>311</td>
<td>Anna Cichonska, Juho Rousu, Pekka Marttinen, Antti J Kangas, Pasi Soininen, Terho Lehtimäki, Olli T Raitakari, Marjo-Riitta Järvelin, Veikko Salomaa, Mika Ala-Korpela, Samuli Ripatti and Matti Pirinen.<strong> Multivariate Meta-Analysis of Genome-Wide Association Studies using Univariate Summary Statistics</strong></td>
</tr><tr><td>312</td>
<td>Michał Ciach and Anna Muszewska.<strong> Searching for Horizontal Gene Transfer events in fungal genomes</strong></td>
</tr><tr><td>313</td>
<td>Marcin Świstak, Magdalena Szuplewska and Dariusz Bartosik.<strong> Genomic analysis of plasmids pJ2P1 and pK8P1 from Antarctic strains of Pseudomonas spp.</strong></td>
</tr><tr><td>314</td>
<td>Chang Pyo Hong, Dongsung Ryu, Sanghoon Song and Junsu Ko.<strong> Genome-wide Analysis of DNA Methylation in Human Breast Cancer Cells through Targeted Methyl-Seq</strong></td>
</tr><tr><td>315</td>
<td>Agata Dziedzic and Anna Muszewska.<strong> Serine protease evolution in fungi with variable lifestyles</strong></td>
</tr><tr><td>316</td>
<td>Konrad Dębski, Katja Kobow, Anna Bot, Noora Huusko, Mark Ziemann, Antony Kaspi, Assam El-Osta, Ingmar Blumcke, Asla Pitkänen and Katarzyna Łukasiuk.<strong> Comparison of DNA methylation events in three animal models of temporal lobe epilepsy</strong></td>
</tr><tr><td>317</td>
<td>Anna Papiez, Michal Marczyk, Andrzej Polanski and Joanna Polanska.<strong> IDENTIFYING BATCH EFFECTS IN HIGH-THROUGHPUT BIOLOGICAL DATA USING DYNAMIC PROGRAMMING BASED APPROACH</strong></td>
</tr><tr><td>318</td>
<td>Witold Rudnicki, Teresa Mroczek and Paweł Cudek.<strong> Amino Acid Properties Conserved in Molecular Evolution</strong></td>
</tr><tr><td>319</td>
<td>Yingying Wei, Toyoaki Tenzen and Hongkai Ji.<strong> Joint Analysis of Differential Gene Expression in Multiple Studies using Correlation Motifs</strong></td>
</tr><tr><td>320</td>
<td>Wojciech Labaj and Andrzej Polanski.<strong> Gene signature optimization based on improved method for enrichment analysis</strong></td>
</tr><tr><td>321</td>
<td>Alon Diament and Tamir Tuller.<strong> Accounting for biases in riboprofiling data indicates a correlation between typical codon decoding rates and tRNA copy numbers</strong></td>
</tr><tr><td>322</td>
<td>Mateusz Kurcinski, Michal Jamroz, Maciej Blaszczyk, Andrzej Kolinski and Sebastian Kmiecik.<strong> CABS-dock web server for flexible docking of peptides to proteins without prior knowledge of the binding site</strong></td>
</tr><tr><td>323</td>
<td>Michał Startek, Arnaud Le Rouzic and Gambin Anna.<strong> Optimal mutation rates for adaptation in Fisher's geometric model with environmental stress.</strong></td>
</tr><tr><td>324</td>
<td>Uzma Mahmood and Zaheer Ul-Haq.<strong> Docking based 3D-QSAR studies applied at the BRAF inhibitors to understand the binding mechanism</strong></td>
</tr><tr><td>325</td>
<td>Joanna Zyla, Marzena Dolbniak, Christophe Badie, Ghazi Alsbeih and Joanna Polanska.<strong> Trend control focused integration in modeling of genotype-phenotype interactions</strong></td>
</tr><tr><td>326</td>
<td>Justyna Kotas, Christophe Badie, Serge Candéias and Joanna Polańska.<strong> Usage of bootstrap sampling in comparing biological diversity of differently sized data samples</strong></td>
</tr><tr><td>327</td>
<td>Prashanthi Dharanipragada and Nita Parekh.<strong> Copy Number Variation Analysis of Diffuse Large B-Cell Lymphoma (DLBCL) Subtypes</strong></td>
</tr><tr><td>328</td>
<td>Shah Md. Shahik and Ismot Ara.<strong> In Silico Homology Modeling, Functional annotation and Molecular Docking Studies of hypothetical proteins of Aspergillus fumigatusAf293.</strong></td>
</tr><tr><td>329</td>
<td>Damian Wójtowicz, Fedor Kouzine, Arito Yamane, Craig J. Benham, Rafael C. Casellas, David Levens and Teresa M. Przytycka.<strong> Genome-wide mapping and computational analysis of non-B DNA structures in vivo</strong></td>
</tr><tr><td>330</td>
<td>Jaroslaw Paszek and Paweł Górecki.<strong> Episode clustering problems for unrooted gene trees</strong></td>
</tr><tr><td>331</td>
<td>Agnieszka Mykowiecka and Paweł Górecki.<strong> Towards more realistic tree reconciliation</strong></td>
</tr><tr><td>332</td>
<td>Rafal Jakubowski, Lukasz Peplowski and Wieslaw Nowak.<strong> On the nature of potential energy surface in autism related synaptic protein NRX-NLG complex</strong></td>
</tr><tr><td>333</td>
<td>Anil Raj, Sidney Wang, Heejung Shim, Yang Li, Matthew Stephens, Yoav Gilad and Jonathan Pritchard.<strong> Accurate inference of thousands of novel translated open reading frames and dually coded regions using ribosome footprinting data.</strong></td>
</tr><tr><td>334</td>
<td>Magdalena A. Machnicka, Katarzyna H. Kaminska, Stanislaw Dunin-Horkawicz and Janusz M. Bujnicki.<strong> Sequence-structure-function relationships in the GmrSD family of Type IV restriction enzymes</strong></td>
</tr><tr><td>335</td>
<td>Samuel Handelman, Michal Seweryn, Andrzej Kloczkowski and Wolfgang Sadee.<strong> Building a consensus co-expression network for CNS genes relevant to pharmacogenomics</strong></td>
</tr><tr><td>336</td>
<td>Agnieszka Blachowicz, Soile Tapio, Zarko Barjaktarovic, Rafi Benotmane, Rosemary Finnon, Christophe Badie, Simon Bouffler and Joanna Polanska.<strong> DATA DRIVEN ESTIMATION OF CUTOFFs IN SEARCHING FOR DIFFERENTIALLY EXPRESSED PROTEINS</strong></td>
</tr><tr><td>337</td>
<td>Katherine Hartmann and Michał Seweryn.<strong> Variability of protein coding and non-coding co-expression patterns</strong></td>
</tr><tr><td>338</td>
<td>Franciszek Binczyk, Cristian Weber, Michael Götz, Bram Stjeltjes, Klaus Maier-Hein, Hans-Peter Meinzer, Rafal Tarnawski, Barbara Bobek-Billewicz and Joanna Polanska.<strong> Determination of high-grade brain tumours internal structure based on magnetic resonance diffusion imaging and signal decomposition to Gaussian mixture model.</strong></td>
</tr><tr><td>340</td>
<td>Michael Schwemmer and Michal Seweryn.<strong> Maximum Renyi Entropy models with higher order interactions in genomics</strong></td>
</tr><tr><td>341</td>
<td>Imene Boudellioua, Rabie Saidi, Maria Martin and Victor Soloviev.<strong> Association Rule Mining for Metabolic Pathway Prediction</strong></td>
</tr><tr><td>342</td>
<td>Mateusz Garbulowski and Andrzej Polański.<strong> Methods of identification and analysis of errors in shotgun sequencing data</strong></td>
</tr><tr><td>343</td>
<td>Aleksandra Gruca, Marek Sikora and Łukasz Stypka.<strong> Expert-driven validation, interpretation and functional description of gene sets.</strong></td>
</tr><tr><td>344</td>
<td>Harriet Mellenius and Måns Ehrenberg.<strong> Modeling of transcriptional accuracy suggests two-step proofreading by RNA polymerase</strong></td>
</tr><tr><td>345</td>
<td>Julia Herman-Izycka and Bartek Wilczynski.<strong> Identifying tissue specific enhancers by sequence and histone modifications - a machine learning approach</strong></td>
</tr><tr><td>346</td>
<td>Mohammed El-Kebir, Layla Oesper, Hannah Acheson-Field and Ben Raphael.<strong> Reconstruction of clonal trees and tumor composition from multi-sample cancer sequencing data</strong></td>
</tr><tr><td>347</td>
<td>Tymoteusz Oleniecki, Maciej Dziubiński, Grzegorz Firlik, Agnieszka Mykowiecka and Paweł Daniluk.<strong> PyDesc: a framework for structural analysis of biopolymers</strong></td>
</tr><tr><td>348</td>
<td>Paweł P. Łabaj and David P. Kreil.<strong> Power and Limitations of RNA-Seq: Findings from the SEQC (MAQC-III) consortium</strong></td>
</tr><tr><td>349</td>
<td>Rafael Zambrano, Michal Jamroz, Agata Szczasiuk, Jordi Pujols, Sebastian Kmiecik and Salvador Ventura.<strong> Aggrescan3D web server for protein aggregation prediction taking into account protein structure and its dynamic fluctuations</strong></td>
</tr><tr><td>350</td>
<td>Agnieszka Mykowiecka, Tymoteusz Oleniecki, Maciej Dziubiński and Paweł Daniluk.<strong> Describing tertiary RNA structure with local spatial motifs</strong></td>
</tr><tr><td>351</td>
<td>Roman Jaksik and Krzysztof Puszyński.<strong> Solvary: a tool for the study of complex intracellular signalling pathways</strong></td>
</tr><tr><td>352</td>
<td>Behrooz Torabi Moghadam, Michal Dabrowski, Bozena Kaminska, Manfred Grabherr and Jan Komorowski.<strong> A multivariate approach predicts genes that change in DNA methylation patterns over age in the human brain</strong></td>
</tr><tr><td>353</td>
<td>Alessandro Mammana and Ho-Ryun Chung.<strong> Improved chromatin segmentation with a probabilistic model for read counts</strong></td>
</tr><tr><td>354</td>
<td>Anna Macioszek, Anna Fogtman, Aleksandra Kwiatkowska, Maciej Kotliński, Roksana Iwanicka-Nowicka, Andrzej Jerzmanowski, Marta Koblowska and Bartek Wilczyński.<strong> Bioinformatic analysis of ChIP-seq data on epigenetic response to salt stress in Arabidopsis thaliana</strong></td>
</tr><tr><td>355</td>
<td>Aleksandra Jarmolińska and Joanna I Sulkowska.<strong> Knot position within a protein, a DCA approach</strong></td>
</tr><tr><td>356</td>
<td>Lukasz Kozlowski and Janusz Bujnicki.<strong> Identification of potential protelomerases and their target sites in publicly available genomes. Phylogenetics analysis</strong></td>
</tr><tr><td>357</td>
<td>Joanna Sulkowska.<strong> KnotProt: a database of proteins with knots and slipknots</strong></td>
</tr><tr><td>358</td>
<td>Sandra Smieszek and Bartlomiej Przychodzen.<strong> How progressive cis element combinations classify conserved orthogonal plant circadian transcriptional modules</strong></td>
</tr><tr><td>359</td>
<td>Michał Boniecki, Grzegorz Łach, Konrad Tomala, Wayne Dawson, Paweł Łukasz, Tomasz Sołtysiński, Kristian Rother and Janusz Bujnicki.<strong> SimRNA: a coarse-grained method for RNA folding simulations and 3D structure prediction.</strong></td>
</tr><tr><td>360</td>
<td>Aleksandra Dawid and Dominik Gront.<strong> Cartesian Rotamers Library for protein structure prediction.</strong></td>
</tr><tr><td>361</td>
<td>Michał Woźniak, Limsoon Wong and Jerzy Tiuryn.<strong> Tree-generalized hypergeometric score for detection of drug resistance-associated mutations</strong></td>
</tr><tr><td>362</td>
<td>Tomasz Jetka, Tomasz Winarski and Michał Komorowski.<strong> Measuring information transmission from single-cell heterogenous dynamical responses</strong></td>
</tr><tr><td>363</td>
<td>Alex Salo, Steven Gonzales, Kathy Grant and Erich Baker.<strong> Automated predication of drinking categories in monkeys undergoing chronic alcohol self-administration</strong></td>
</tr><tr><td>364</td>
<td>Young Park, Sunhye Park, Yoon Cho, Kiejung Park and Insong Koh.<strong> A pilot study for the construction of Korean-specific exome-variome database</strong></td>
</tr><tr><td>365</td>
<td>Linda K. Sundermann, Amit G. Deshwar, Quaid Morris and Gunnar Rätsch.<strong> Onctopus: A New Model for Subclonal Composition Reconstruction</strong></td>
</tr><tr><td>366</td>
<td>Norbert Dojer, Abhishek Mitra, Yea-Lih Lin, Anna Kubicka, Magdalena Skrzypczak, Krzysztof Ginalski, Philippe Pasero and Maga Rowicka.<strong> Computational detection of DNA double-stranded breaks with nucleotide resolution using deep sequencing data</strong></td>
</tr><tr><td>367</td>
<td>Prajwal Devkota, Bonnie Kirkpatrick, Susan Blanton and Alexandre Bouchard-Cote.<strong> Correcting for Cryptic Relatedness in Genome-Wide Association Studies</strong></td>
</tr><tr><td>368</td>
<td>Junil Kim, Jeong-Rae Kim and Seong-Jin Kim.<strong> Mathematical model of human gene regulatory network</strong></td>
</tr><tr><td>369</td>
<td>Maga Rowicka, Norbert Dojer, Ji Li, Yea-Lih Lin, Magdalena Skrzypczak, Anna Kubicka, Krzysztof Ginalski and Philippe Pasero.<strong> Inferring direction of replication fork and mechanism of DNA damage using sequencing data</strong></td>
</tr><tr><td>370</td>
<td>Cheng Zhao, Ying Li, Benjamin Haibe-Kains and Anna Goldenberg.<strong> Using cell lines and patient samples to improve patients’ drug response prediction</strong></td>
</tr><tr><td>371</td>
<td>Daria Iakovishina, Mireille Regnier, Valentina Boeva and Emmanuel Barillot.<strong> SV-Bay: structural variant detection in cancer genomes using a Bayesian approach with correction for GC-content and read mappability</strong></td>
</tr><tr><td>372</td>
<td>Serghei Mangul, Farhad Hormozdiari, Elizabeth Tseng, Alex Zelikovsky and Eleazar Eskin.<strong> (Poster) HapIso : An accurate method for the haplotype-specific isoforms reconstruction from long single-molecule reads</strong></td>
</tr><tr><td>373</td>
<td>Lee Gierke Cathy.<strong> ANALYSIS OF RHYTHMS USING R: CHRONOMICS ANALYSIS TOOLKIT (CAT)</strong></td>
</tr><tr><td>374</td>
<td>Wolfgang Kopp and Martin Vingron.<strong> A Bayesian model for the number of motif hits in finite double-stranded DNA sequences</strong></td>
</tr><tr><td>375</td>
<td>Ilona Foik, Irina Tuszynska, Marcin Feder, Elżbieta Purta and Janusz Bujnicki.<strong> Novel inhibitors of ErmC’ methyltransferase responsible for resistance to MLSB antibiotics</strong></td>
</tr><tr><td>377</td>
<td>Jonas A. Sibbesen, Lasse Maretty and Anders Krogh.<strong> Probabilistic genotyping without alignment</strong></td>
</tr><tr><td>378</td>
<td>Mohammadhossein Moeinzadeh, Jun Yang and Martin Vingron.<strong> Haplotype reconstruction for polyploid organisms</strong></td>
</tr><tr><td>379</td>
<td>Bartosz Wojtas, Bartlomiej Gielniewski, Marta Maleszewska, Mateusz Bujko, Janusz Siedlecki, Katarzyna Kotulska, Wieslawa Grajkowska and Bozena Kaminska.<strong> Targeted enrichment sequencing of the large genomic region in glioblastoma</strong></td>
</tr><tr><td>380</td>
<td>Isaac Joseph, Shannon McCurdy, Joseph F. Costello and Lior Pachter.<strong> A method for combining multiple genomic and clinical datatypes to predict recurrence grade in gliomas</strong></td>
</tr><tr><td>382</td>
<td>Matt Huska and Martin Vingron.<strong> The DNA sequence features of non-methylated islands across six vertebrates</strong></td>
</tr><tr><td>383</td>
<td>Dominik Gront.<strong> Three dimensional threading for protein structure modelling</strong></td>
</tr><tr><td>385</td>
<td>Przemek Biecek.<strong> A more powerful test for identification of differentially methylated regions</strong></td>
</tr><tr><td>386</td>
<td>Sergiy Ancherbak, Ercan Kuruoglu and Martin Vingron.<strong> Time-varying Gene Interaction Network Modeling by Sequential Monte Carlo</strong></td>
</tr></tbody></table></div></div></div>
  
  
</article>
              </div>

      <div id="navigation">
        <nav id="main-menu" role="navigation" tabindex="-1">
                  </nav>
      </div>

      
              <aside class="sidebars">
            <section class="region region-sidebar-first column sidebar">
    <div id="block-system-main-menu" class="block block-system block-menu first last odd" role="navigation">

      
  <ul class="menu"><li class="menu__item is-leaf first leaf"><a href="../index.html" class="menu__link">Home</a></li>
<li class="menu__item is-leaf leaf"><a href="6.html" class="menu__link">Program</a></li>
<li class="menu__item is-leaf leaf"><a href="3.html" class="menu__link">Venue</a></li>
<li class="menu__item is-collapsed collapsed"><a href="24.html" class="menu__link">Satellite Meetings</a></li>
<li class="menu__item is-leaf leaf"><a href="46.html" class="menu__link">Affiliated events</a></li>
<li class="menu__item is-leaf leaf"><a href="5.html" class="menu__link">Registration</a></li>
<li class="menu__item is-leaf leaf"><a href="7.html" class="menu__link">Travel &amp; Accomodation</a></li>
<li class="menu__item is-leaf leaf"><a href="11.html" class="menu__link">Accepted papers</a></li>
<li class="menu__item is-active-trail is-leaf leaf active-trail"><a href="48.html" class="menu__link is-active-trail active-trail active">Accepted Posters</a></li>
<li class="menu__item is-leaf leaf"><a href="18.html" title="" class="menu__link">Keynote Speakers</a></li>
<li class="menu__item is-leaf leaf"><a href="8.html" class="menu__link">Call for posters</a></li>
<li class="menu__item is-leaf leaf"><a href="9.html" class="menu__link">Call for highlights</a></li>
<li class="menu__item is-leaf leaf"><a href="40.html" class="menu__link">Travel fellowships</a></li>
<li class="menu__item is-leaf leaf"><a href="12.html" class="menu__link">About RECOMB</a></li>
<li class="menu__item is-leaf leaf"><a href="16.html" class="menu__link">Committees</a></li>
<li class="menu__item is-leaf leaf"><a href="15.html" class="menu__link">Sponsors</a></li>
<li class="menu__item is-leaf leaf"><a href="28.html" title="" class="menu__link">Call for papers</a></li>
<li class="menu__item is-leaf last leaf"><a href="22.html" class="menu__link">Contact</a></li>
</ul>
</div>
  </section>
                  </aside>
      
    </div>

  </div>

  <div id="page-footer">
    <div id="page-footer-inner">
        <footer id="footer" class="region region-footer">
    <div id="block-block-1" class="block block-block first last odd">

      
  <p>DESIGN: <a href="http://podpunkt.pl" target="_blank">Podpunkt</a> | PRODUCTION: <a href="http://intools.pl" target="_blank">inTOOLS</a></p>

</div>
  </footer>
    </div>
  </div>
  </div>  </body>
</html>
